projects/WTI-UI/src/app/modules/shared/components/app-header/app-header.component.scss

```diff
@@ -14,6 +14,16 @@ header {
   .teamid-container {
     color: white;
     font-size: 1.5rem;
+    padding-left: 8px;
+    padding-right: 8px;
+  }
+  
+  /* Set the container holding the contest clock value(s) to medium, white font */
+  .clock-container {
+	    color: white;
+    font-size: 1.0rem;
+    padding-left: 8px;
+    padding-right: 8px;
```

Comment:
I have some concerns about how these pixel paddings would look in different window sizes. Did you play around with different window size and different pixel counts? Perhaps we could use relative units such as em/ rem, vw, just like you use it for font-size. But this is not necessarily problematic, there are cases where pixels are better.

Reply:
I did try it using various window sizes, and I didn't spot any problems.  Do you have a specific example of a platform and window size which shows a problem?

Regarding the "pixel paddings", I didn't create those; they came from a CSS file I got from @johnbrvc .   But I haven't observed any issues with it (I tested it in both Chrome and Edge, using various window sizes).  Again, do you have an example of a specific situation where a problem is demonstrated?

Reply:
Doesn't seem to be an issue.

Reply:
Based on @kkarakas comment below ("Doesn't seem to be an issue.") I'm marking this Resolved.